app/templates/components/job-controls.hbs

```diff
@@ -6,16 +6,20 @@
     {{#if error}}
       {{#bs-alert type="danger"}}<span class="error-message">{{error}}</span>{{/bs-alert}}
     {{/if}}
-    {{#if job.isNew}}
+    {{#if job.isAuthorized}}
```

Comment:
We haven't ever gone back to NEW. 
If a job fails it goes to CANCEL or ERROR state.
From both of those you should be able to restart.